bedrock/firefox/templates/firefox/facebookcontainer/index.html

```diff
@@ -0,0 +1,123 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% add_lang_files "firefox/facebookcontainer" %}
```

Comment:
As we aren't localizing yet, will this confuse L10n? (Ignore if this has already been discussed with the L10n team.)

Reply:
I'm not sure of the longer-term l10n plan or how long this page will live. I'll remove the lang file and if we decide to localize it later we can always add it back.